pkgs/applications/video/wivrn/default.nix

```diff
@@ -0,0 +1,235 @@
+{
+  config,
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchFromGitLab,
+  applyPatches,
+  autoAddDriverRunpath,
+  avahi,
+  bluez,
+  boost,
+  cjson,
+  cli11,
+  cmake,
+  cudaPackages ? { },
+  cudaSupport ? config.cudaSupport,
+  dbus,
+  # depthai
+  doxygen,
+  eigen,
+  elfutils,
+  ffmpeg,
+  freetype,
+  git,
+  glib,
+  glm,
+  glslang,
+  gst_all_1,
+  harfbuzz,
+  hidapi,
+  # leapsdk
+  # leapv2
+  libGL,
+  libX11,
+  libXrandr,
+  libbsd,
+  libdrm,
+  libdwg,
+  libjpeg,
+  libmd,
+  libpulseaudio,
+  librealsense,
+  libsurvive,
+  libunwind,
+  libusb1,
+  libuvc,
+  libva,
+  makeDesktopItem,
+  nix-update-script,
+  nlohmann_json,
+  onnxruntime,
+  opencv4,
+  openhmd,
+  openvr,
+  openxr-loader,
+  orc,
+  # percetto
+  pipewire,
+  pkg-config,
+  python3,
+  qtbase,
+  qttools,
+  SDL2,
+  shaderc,
+  spdlog,
+  systemd,
+  udev,
+  vulkan-headers,
+  vulkan-loader,
+  wayland,
+  wayland-protocols,
+  wayland-scanner,
+  wrapQtAppsHook,
+  x264,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wivrn";
+  version = "0.20";
+
+  src = fetchFromGitHub {
+    owner = "wivrn";
+    repo = "wivrn";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-mxvfwp/9CUoc6tU3KW257qlpXEZu7tK33jxn1TjAZYc=";
+  };
+
```

Comment:
```suggestion

        patches = [
          (pkgs.fetchurl {
            url = "https://github.com/WiVRn/WiVRn/commit/be95ec84b56794ca8d97bb8b21dec6443ba57bc4.diff";
            sha256 = "1tNQ57XIki0bomNG5xg+/4YbgKanrM5LOQvvCmZQFhw=";
          })
        ];

```

Reply:
This fixes the build for me :)

Reply:
I'm going to see if they publish a release with a patch before I do it myself

Reply:
Alrighty